sdv/_utils.py

```diff
@@ -450,3 +450,37 @@ def _is_numerical(value):
         return is_integer(value) or is_float(value)
     except Exception:
         return False
+
+
+def _prepare_data_visualization(data, metadata, column_names, sample_size):
```

Comment:
minor: we could put this in a _utils file inside the evaluation subpackage just to be more organized